images/linux/scripts/installers/1604/kubernetes-tools.sh

```diff
@@ -9,10 +9,10 @@ source $HELPER_SCRIPTS/document.sh
 source $HELPER_SCRIPTS/apt.sh
 
 ## Install kubectl
-apt-get install -y apt-transport-https
+apt-get install -y apt-transport-https gnupg2
```

Comment:
Maybe we can move these installations to basic.sh scripts to install it at the very beginning?

Reply:
Done